roles/php/tasks/main.yml

```diff
@@ -1,21 +1,29 @@
 ---
 - name: Install Epel first
-  yum: 
+  package:
     name: epel-release
     state: present
 
+# at present we need one key
```

Comment:
I don't think I understand this comment. It's clear that we currently indeed install one, but it sounds like there should be a rule this exception proves, and I can't figure it out.

If we expect to need more keys at a later time or something, and that needs to be made known now, we need a more explicit comment. If it's here just to say that we loop over one key, then I'd say that the comment can be removed altogether.